moesif_rack.gemspec

```diff
@@ -1,15 +1,16 @@
 Gem::Specification.new do |s|
   s.name = 'moesif_rack'
-  s.version = '1.4.13'
+  s.version = '1.4.14'
   s.summary = 'moesif_rack'
   s.description = 'Rack/Rails middleware to log API calls to Moesif API analytics and monitoring'
   s.authors = ['Moesif, Inc', 'Xing Wang']
   s.email = 'xing@moesif.com'
   s.homepage = 'https://moesif.com'
   s.license = 'Apache-2.0'
   s.add_development_dependency('test-unit', '~> 3.1', '>= 3.1.5')
-  s.add_dependency('moesif_api', '~> 1.2', '>= 1.2.12')
-  s.required_ruby_version = '~> 2.0'
+  s.add_dependency('moesif_api', '~> 1.2', '>= 1.2.14')
+  s.add_dependency('addressable', '~> 2.8.0')
```

Comment:
Please change the addressable's version dependency to `>=2.8.0`

Reply:
![image](https://user-images.githubusercontent.com/25126842/152091218-d6d86876-bcae-403b-b37c-880e9fe8e856.png)
We will get this warning after change it to `>= 2.8.0`, is that acceptable?

Reply:
Thanks  @XinyuLuan . No, that's not acceptable. Let's keep addressable as it and change the rest.